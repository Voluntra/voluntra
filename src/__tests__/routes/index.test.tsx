import renderer, { ReactTestRendererJSON } from "react-test-renderer";
import App from "../../app";

describe("<App />", () => {
  it("has at least 1 child", () => {
    const tree = renderer.create(<App />).toJSON() as ReactTestRendererJSON;
    expect(tree.children?.length).toBeGreaterThan(0);
  });
});
