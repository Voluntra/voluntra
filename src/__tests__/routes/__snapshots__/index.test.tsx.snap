// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders the Dashboard correctly 1`] = `
<View>
  <Text
    style={
      [
        {
          "color": "#E0E0E0",
        },
      ]
    }
  >
    You're currently in the dashboard!
  </Text>
</View>
`;

exports[`<App /> renders the Discover correctly 1`] = `
<View>
  <Text>
    You're currently in the discover section!
  </Text>
</View>
`;

exports[`<App /> renders the Settings correctly 1`] = `
<RNCSafeAreaView
  edges={
    {
      "bottom": "additive",
      "left": "additive",
      "right": "additive",
      "top": "additive",
    }
  }
>
  <Text>
    You're currently on the settings page!
  </Text>
</RNCSafeAreaView>
`;

exports[`renders correctly 1`] = `
<View>
  <Text>
    Voluntra's humble beginnings
  </Text>
</View>
`;
